webpack.test.config.js

```diff
@@ -0,0 +1,78 @@
+const helpers = require('./config/helpers');
```

Comment:
I don't think we're using this file, we're only using the `.config.dev.ts` file.
You can remove this one.